packages/dds/tree/src/test/feature-libraries/sequence-field/utils.ts

```diff
@@ -871,10 +872,8 @@ function newCrossFieldTable<T = unknown>(): CrossFieldTable<T> {
 			value: T,
 			invalidateDependents: boolean,
 		) => {
-			if (
-				invalidateDependents &&
-				getFromRangeMap(getQueries(target).get(revision) ?? [], id, count) !== undefined
-			) {
+			const rangeMap = getMap(target).get(revision) ?? new RangeMap<T>();
+			if (invalidateDependents && rangeMap.get(id, count) !== undefined) {
```

Comment:
`rangeMap.get(id, count) !== undefined` will always be true. Did you mean to check for `rangeMap.get(id, count).value !== undefined`?

Reply:
Thanks. There was a also bug here that was covered up by this always returning true. We should be checking the query set (`getQueries`), not the data map (`getMap`).